backend/btrixcloud/webhooks.py

```diff
@@ -183,34 +161,53 @@ async def create_item_created_notification(self, crawl_id: str):
         await self.send_notification(org, notification)
 
         if crawl.collections:
-            for coll_id in crawl.collections:
-                await self.create_added_removed_collection_notification(
+            for coll_id in crawl.collectionIds:
+                await self.create_added_to_collection_notification(
                     crawl_ids=[crawl_id], coll_id=coll_id, org=org
                 )
 
-    async def create_added_removed_collection_notification(
+    async def create_added_to_collection_notification(
```

Comment:
could perhaps still have this as one function, or have them call same function with different `event=` and `body=` params, though maybe clearer this way - just thinking of avoiding unnecessary duplication

Reply:
Hm, actually, yeah I like the idea of them both calling the same function under the hood. Will do that now.

Reply:
Done